src/main/java/net/datafaker/providers/base/Image.java

```diff
@@ -3,97 +3,193 @@
 
 import javax.imageio.ImageIO;
 import java.awt.Color;
-import java.awt.Graphics2D;
+import java.awt.*;
 import java.awt.image.BufferedImage;
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.util.Base64;
 
 import static java.awt.Color.WHITE;
+import static net.datafaker.providers.base.Image.ImageType.BMP;
+import static net.datafaker.providers.base.Image.ImageType.GIF;
+import static net.datafaker.providers.base.Image.ImageType.JPEG;
+import static net.datafaker.providers.base.Image.ImageType.PNG;
+import static net.datafaker.providers.base.Image.ImageType.SVG;
+import static net.datafaker.providers.base.Image.ImageType.TIFF;
 
 /**
- * Generates base64 encoded PNG, GIF, JPG and SVG images.
+ * Generates base64 encoded raster and vector images.
  *
  * @since 2.3.0
  */
 public class Image extends AbstractProvider<BaseProviders> {
 
-    private static final int WIDTH = 256;
-    private static final int HEIGHT = 256;
-    private static final int BOX_SIZE = WIDTH / 8;
+    private static final int DEFAULT_WIDTH = 256;
+    private static final int DEFAULT_HEIGHT = 256;
+
+    public enum ImageType {
+        // See https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types
+        BMP("image/bmp"),
+        GIF("image/gif"),
+        JPEG("image/jpeg"),
+        PNG("image/png"),
+        SVG("image/svg+xml"),
+        TIFF("image/tiff");
```

Comment:
do  we really need `image/` prefix for every item?
May be just add it once in `get` method?

Reply:
Well, no, we wouldn't really need it, but the field is called mimetype, which is why you have of these duplication, but without the `image/` part it wouldn't be a mimetype, it would be..... ?

Reply:
I guess if you replace constructor with
```java
 ImageType(String type) {
      this.mimeType = "images/" + type;
 }     
 ```
 the field would be still `mimeType` however less duplication